build.gradle

```diff
@@ -27,12 +27,11 @@ repositories {
     maven { url 'http://repo.spring.io/release'}
 }
 
-
 dependencies {
 
-    ext.dropwizardVersion = '0.8.1'
+    ext.dropwizardVersion = '0.8.4'
     ext.springIoPlatformVersion = '1.1.2.RELEASE'
-    ext.jerseyVersion = '1.19'
+    ext.jerseyVersion = '2.22.1'
```

Comment:
This is the big issue with Fallwizard right now: we explicitly use jersey version 1.19 in our code.  2.x is not backwards-compatible and none of us use Fallwizard any longer, so there's been no burning need to take that code and port it to 2.x.

If you all are still using Fallwizard... and you've already ported it, send me a pull request and I'll accept it.